### PR TITLE
Modul 159 remove peer dependencies

### PR DESCRIPTION
**Pull request by chuckmah**
## `@ulaval/modul-components`
# PR Checklist

- [x] Provide a small description of the changes introduced by this PR

Removed some peerDependencies and clean up of dependencies in package.json 

Summary of how dependecies should behavior :

`dependencies` is used for required dependencies (ie. non optional third party)  **that should be installed transitivitly**. 

`peerDependecies` is used for required dependencies **that should not be installed separately** (such as vue)

`devDependecies` should include everything else such as building/testing/sandbox /meta **that should not be installed transitivitly**

- [x] Include links to issues

- [ ] Openshift deployment requested

- [X] Include this section in the release notes

Some transitives dependencies were removed from modul and should now be installated manually via npm 

If something is missing run the following command to ensure all dev deps properly installed in your project.

`npm install --save-dev @types/sprintf-js  css-loader node-sass postcss-loader sass-loader style-loader svg-inline-loader vue-class-component vue-property-decorator vue-template-compiler vue-template-loader
`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,52 +69,39 @@
...
-        "url-loader": "^0.6.2",
-        "vue": "^2.5.9",
+        "filesize": "^3.6.0",
         "vue-touch": "^2.0.0-beta.4"
     },
+    "peerDependencies": {
+        "vue": "^2.5.9"
+    },
     "devDependencies": {
         "@types/filesize": "^3.6.0",
```
Comment (vidal7) [inline package.json]: Je pense que @types/filesize devrait être une dependencies car filesize est dans les dependencies

Comment (chuckmah) [reply package.json]: On peut l'ajouter aucun problème avec ca.

mais dans l'optique ou on modul-component ne force pas l'utilisation de typescript on devrait pas forcer les @types non?

Comment (vidal7) [reply package.json]: Ouin, tu marques un point! modUL ne devrait pas forcer typescripté

File: package.json [hunk 2]
```diff
@@ -69,52 +69,39 @@
...
+        "zingtouch": "^1.0.6",
         "popper.js": "^1.12.9",
-        "svg-inline-loader": "^0.8.0",
-        "url-loader": "^0.6.2",
-        "vue": "^2.5.9",
+        "filesize": "^3.6.0",
         "vue-touch": "^2.0.0-beta.4"
     },
+    "peerDependencies": {
+        "vue": "^2.5.9"
```
Comment (vidal7) [inline package.json]: Ajouter comme peerDependencies vue-router car m-link génère des router-link

Comment (chuckmah) [reply package.json]: @vidal7 c'est fait

**Pull request merged**